src/test/java/seedu/address/logic/parser/UnlabelCommandParserTest.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_LABEL_DESC;
+import static seedu.address.logic.commands.CommandTestUtil.INVALID_TAG_DESC;
+import static seedu.address.logic.commands.CommandTestUtil.TAG_DESC_CS2103;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_LABEL;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_LABEL_DESC;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_NAME_CS2103;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.UnlabelCommand;
+import seedu.address.model.label.Label;
+import seedu.address.model.tag.TagName;
+
+public class UnlabelCommandParserTest {
```

Comment:
Test for non-empty preamble (parse with a string starting with "unlabel") for 100% coverage?

Test with `new UnlabelCommandParser.parser("unlabel ...")`.

Testing this condition: `!argMultimap.getPreamble().isEmpty()`

Reply:
Added!